Vagrantfile

```diff
@@ -4,13 +4,13 @@
 # Ruby, run unit tests, etc.
 
 Vagrant.configure("2") do |config|
-  config.vm.box = "hashicorp/precise64"
+  config.vm.box = "ubuntu/bionic64"
```

Comment:
This should be `hashicorp/bionic64` now. Thanks!

Reply:
Didn't realize there was a hashicorp box for bionic64. Switched!